opencontext_py/apps/archive/tests.py

```diff
@@ -28,10 +29,15 @@ def setUp(self):
                  'rdflib_jsonld.parser',
                  'JsonLDParser')
         # self.graph = Graph()
-        self.proj_context_uri = 'http://127.0.0.1:8000/contexts/projects/3.json'
+        # self.proj_context_uri = 'http://127.0.0.1:8000/contexts/projects/3FAAA477-5572-4B05-8DC1-CA264FE1FC10.json'
+        self.proj_context_uri = '/contexts/projects/3FAAA477-5572-4B05-8DC1-CA264FE1FC10.json'
         self.context_file = settings.STATIC_IMPORTS_ROOT + '3-context.json'
         self.data_file = settings.STATIC_IMPORTS_ROOT + 'dt-bone.json'
-        self.context_str = self.request_json_str(self.proj_context_uri)
+        #self.context_str = self.request_json_str(self.proj_context_uri)
+        client = Client()
+        response = client.get(self.proj_context_uri, follow=True)
+        assert response.status_code in [200,301]
+        self.context_str = response.content
         self.data_str = self.load_json_file_str(self.data_file)
```

Comment:
at this point, the setup still fails because `self.data_str` is still None.

Reply:
do we need to check in dt-bone.json into the STATIC_IMPORTS_ROOT dir? Is there a complication to this?